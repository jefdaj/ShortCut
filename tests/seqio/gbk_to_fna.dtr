gbk6803 # test comment after varname
  = load_gbk "data/Synechocystis-PCC-6803-uid159873.gbk"
fna6803 = # test comment after = sign
  gbk_to_fna gbk6803
result  = fna6803
