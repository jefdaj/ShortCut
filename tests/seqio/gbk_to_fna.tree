.
|-- cache
|   `-- load
|       `-- fb223a32fa.gbk -> $WORKDIR/data/Synechocystis-PCC-6803-uid159873.gbk
|-- exprs
|   |-- gbk_to_fna
|   |   `-- c0ff4985c5.fna
|   |-- load_gbk
|   |   `-- e57645fe99.gbk -> ../../cache/load/fb223a32fa.gbk
|   `-- str
|       `-- 0507961f5b.str -> ../../cache/lines/...
`-- vars
    |-- fna6803.fna -> ../exprs/gbk_to_fna/c0ff4985c5.fna
    |-- gbk6803.gbk -> ../exprs/load_gbk/e57645fe99.gbk
    `-- result -> ../vars/fna6803.fna

7 directories, 7 files
