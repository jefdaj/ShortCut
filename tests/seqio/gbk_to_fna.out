>SYNGTS_0001 YP_005649954.1 sds solanesyl diphosphate synthase
ATGATCTCCACTACCTCCCTGTTTGCCCCCGTTGACCAAGACCTCCGTTTATTAACGGATAATCTCAAGCGGCTTGT...
>SYNGTS_0002 YP_005649955.1 slr0612 hypothetical protein
GTGGGGCGTTTGGATCAAGATAGCGAAGGACTATTGCTGCTCACCAGCAACGGTAAACTTCAGCATCGTTTGGCCCA...
>SYNGTS_0003 YP_005649956.1 slr0613 hypothetical protein
...

